### ISSUE TITLE
 Are there depth images in this dataset?

### ISSUE THREAD

**Issue by modddypan** [zhezh/occlusion_person#1]

**Comment by zhezh** [comment]
Hi @pzxdd , due to depth images take a lot of storage space, we have removed all the depth images after we obtained the occlusion labels.

**Comment by modddypan** [comment]
thanks for response and this excellent datasets!

why i ask? because I want to use depth image to generate point clouds.

so,do you have any plan to release depth images in the future? or provide download link?or any tools to generate depth image from these images(like unrealCV?).

thx again!

**Comment by zhezh** [comment]
@pzxdd the unrealcv tool is open sourced by weichao at https://unrealcv.org/

**Comment by modddypan** [comment]
I am not familiar with unrealcv.o(╥﹏╥)o

Can  I use this tool to generate depth images?

like this:
input:   an rgb image from this datasets ----- 》 use unrealcv ------》output:  corresponding depth image

**Comment by zhezh** [comment]
@pzxdd No, it is not working like this, unrealcv is build on top of unreal which is a game engine. Unrealcv provides interface to capture the depth, rgb and segmentation of the virtual scene in unreal. Just take a look at the demos and description on unrealcv web pages to be familiar with it.

**Comment by modddypan** [comment]
ok,thanks a lot!!!!
